Formula/codequery.rb

```diff
@@ -13,13 +13,18 @@ class Codequery < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "qt"
+  depends_on :python3 => :build
+  depends_on "qt5"
   depends_on "qscintilla2"
 
   def install
+    args = std_cmake_args
+    args << "-DBUILD_QT5=ON"
+    args << "-DQT5QSCINTILLA_LIBRARY=#{Formula["qscintilla2"].opt_prefix}/lib/libqscintilla2.dylib"
```

Comment:
`#{Formula["qscintilla2"].opt_prefix}/lib` can be `#{Formula["qscintilla2"].opt_lib}` though.